service/src/main/java/greencity/service/notification/ManagementNotificationServiceImpl.java

```diff
@@ -45,7 +54,9 @@ public void update(NotificationTemplateDto notificationTemplateDto) {
             .descending());
         Page<NotificationTemplate> notificationTemplates = notificationTemplateRepository.findAll(pageRequest);
         List<NotificationTemplateDto> templateDtoList = notificationTemplates.stream()
-            .map(x -> modelMapper.map(x, NotificationTemplateDto.class)).collect(Collectors.toList());
+            .map(x -> modelMapper.map(x, NotificationTemplateDto.class)
```

Comment:
Try to find better name for variable, better to use name like templateDto

Reply:
I think if i have list of dtos i like to give them name dtolist more than just dto

Reply:
@DimaSannytski I mean rename x

Reply:
.map(x( is dto)->mod.map(x( is entity)) in my opinion it's not the best idea rename x to templatedto

Reply:
agree. please avoid names like "X"